superset-frontend/src/explore/components/controls/FilterBoxItemControl.jsx

```diff
@@ -60,7 +63,35 @@ export default class FilterBoxItemControl extends React.Component {
     this.props.onChange(this.state);
   }
   onControlChange(attr, value) {
-    this.setState({ [attr]: value }, this.onChange);
+    let typedValue = value;
+    const { column: selectedColumnName, multiple } = this.state;
+    if (value && !multiple && attr === 'defaultValue') {
```

Comment:
should `defaultValue` be a string constant somewhere that's used here and in the filter config?

Reply:
added 2 constants, `defaultValue` and `multi` (used by this feature). but there are quite a few other attributes used in many places...